src/components/CytoscapeGraph/EmptyGraphLayout.tsx

```diff
@@ -139,7 +139,7 @@ export default class EmptyGraphLayout extends React.Component<EmptyGraphLayoutPr
           <Title headingLevel="h5" size="lg">
             Empty Graph
           </Title>
-          <EmptyStateBody>Not enough data to generate a graph.</EmptyStateBody>
+          <EmptyStateBody>No traffic data to generate a graph.</EmptyStateBody>
```

Comment:
Ok, perhaps that message can be refined, because that empty message case only happens for the minigraph which is intended to be in the details.

Reply:
Oh right, we already have a more robust message for the when the main graph is Empty.  The "selected namespaces" part doesn't make any sense but I'd still like to indicate that it could just be for this rime period, so how about:

"No graph traffic for the time period."